### ISSUE TITLE
GuzzleHttp version conflicting with other plugins for lack of centralized libraries

### ISSUE THREAD

**Issue by drzraf** [trilbymedia/grav-plugin-login-oauth2#41]
I would like to bring your attention on https://github.com/stephan-strate/grav-plugin-logger-channels/issues/4
- `grav-plugin-login-oauth2 : guzzlehttp/guzzle : 6.5.4,  GuzzleHttp::choose_handler()`
- `grav-plugin-logger-channels : guzzlehttp/guzzle : 7.2.0, GuzzleHttp\Utils::chooseHandler()`

Both do `return require __DIR__ . '/vendor/autoload.php';` (and corresponding `spl_autoload_register` with different entries for `GuzzleHttp`) in a _first-win_ fashion which make both plugins de facto incompatibles.

I'm pretty sure it's not the first time such an issue arise since it's deemed to happen if each plugin manages its own set of dependencies (which are likely redundant across a Grav installation).

**Comment by mahagr** [comment]
Yes, this is the downside of plugins using vendor libraries. Sometimes they even conflict with Grav itself.
